geomstats/sampling/lognormal.py

```diff
@@ -0,0 +1,73 @@
+import geomstats.geometry.spd_matrices as spd
+import geomstats.backend as gs
+
+class LogNormal:
+    """ LogNormal Sampler 
```

Comment:
NIT: Could you just review our conventions for docstrings? For example, there needs to be a period at the end of end function description, and parameter description (below), as well as a space before the ":" in the parameter descriptions. Thank you! 🙏